modules/face/samples/sample_config_file.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0"?>
```

Comment:
Somewhere this XML file will need to be documented accurately, else tons of people will not understand what each parameter is doing. Maybe add comments inside of it?